hooks/install-hooks.sh

```diff
@@ -3,14 +3,20 @@
 HOOK_NAMES="pre-commit"
 HOOK_DIR=$(git rev-parse --show-toplevel)/.git/hooks
 INSTALL_DIR=$(git rev-parse --show-toplevel)/hooks
+COLOR_GREEN=`tput setaf 2`
+COLOR_RESET=`tput sgr0`
 
 for hook in $HOOK_NAMES; do
+    echo -n "Installing $hook hook..."
     # If the hook already exists, is executable, and is not a symlink
     if [ ! -h $HOOK_DIR/$hook -a -x $HOOK_DIR/$hook ]; then
+        echo -n " Hook already exists..."
         mv $HOOK_DIR/$hook $HOOK_DIR/$hook.local
```

Comment:
It would be a great idea to let the user know that we are saving a backup and _where_:

` Saved old hook backup at .git/hooks/${hook}.local...`

However, I've noticed that the `if` condition doesn't really work, so for testing you'll need to 'hack' the `if`.